### PR TITLE
Include information on article authors in footnote.

### PR DESCRIPTION
**Pull request by petterreinholdtsen**

### CODE REVIEW

File: docs/src/motion/pid-theory.adoc [hunk 1]
```diff
@@ -163,8 +163,8 @@ Effects of increasing parameters
 Another tuning method is formally known as the 'Ziegler-Nichols
 method', introduced by John G. Ziegler and Nathaniel B. Nichols
 footnote:[Introduced in the 1942 paper Optimum Settings for Automatic
-Controllers, link:https://doi.org/10.1115/1.2899060[DOI
-10.1115/1.2899060] also
+Controllers by J. G. Ziegler and N. B. Nichols,
+link:https://doi.org/10.1115/1.2899060[DOI 10.1115/1.2899060] also
```
Comment (hansu) [inline docs/src/motion/pid-theory.adoc:167]: What does "in the 1942 paper" mean here?

**Comment by petterreinholdtsen** [comment]
[Hans Unzner]
> What does "in the 1942 paper" mean here?

The article/paper is from 1942.  The year was copied from a reference in
<URL: https://en.wikipedia.org/wiki/Ziegler%E2%80%93Nichols_method >.  I
am not quite sure why the DOI link uses 1993 as the year.  Perhaps it
was the year of the reprint?

-- 
Happy hacking
Petter Reinholdtsen

**Comment by hansu** [comment]
Ah okay. 

What about this format? the word "introduced" is a bit duplicated here.
```
footnote:[Ziegler, J.G & Nichols, N. B. _Optimum settings for automatic controllers_. Transactions of the ASME, 1942. link:https://web.archive.org/web/20170918055307/http://staff.guilan.ac.ir/staff/users/chaibakhsh/fckeditor_repo/file/documents/Optimum%20Settings%20for%20Automatic%20Controllers%20(Ziegler%20and%20Nichols,%201942).pdf[PDF]]
```

**Comment by petterreinholdtsen** [comment]
[Hans Unzner]
> What about this format? the word "introduced" is a bit duplicated here.
> ```
> footnote:[Ziegler, J.G & Nichols, N. B. _Optimum settings for automatic controllers_. Transactions of the ASME, 1942. link:https://web.archive.org/web/20170918055307/http://staff.guilan.ac.ir/staff/users/chaibakhsh/fckeditor_repo/file/documents/Optimum%20Settings%20for%20Automatic%20Controllers%20(Ziegler%20and%20Nichols,%201942).pdf[PDF]]
> ```

This format lack the DOI, which is vital to uniquely identify the
article.  Beside the missing DOI, it look fine with me.  Suspect the
other foot notes should have a similar and consistent format.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by petterreinholdtsen** [comment]
I adjusted the footnote formatting to look like this:

  Ziegler, J. G. and Nichols, N. B. (1942), Optimum Settings for
  Automatic Controllers, Transactions of the ASME, DOI 10.1115/1.2899060
  and The Internet Archive.

This bring it more in line with traditional reference formatting while
keeping both links available.

-- 
Happy hacking
Petter Reinholdtsen

**Pull request merged**